<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kmessagebox</class>
 <widget class="QWidget" name="Kmessagebox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>158</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>388</width>
    <height>158</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>388</width>
    <height>158</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>120</y>
     <width>391</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pButtonOk">
      <property name="minimumSize">
       <size>
        <width>74</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>74</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pButtonCancel">
      <property name="minimumSize">
       <size>
        <width>74</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>74</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>397</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>12</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>22</number>
    </property>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>48</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="MessageIcon">
      <property name="minimumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>38</width>
        <height>38</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="MessageContent">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputContent">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
